Fixes

elf/librtld.os: In function `_dl_start_final':
elf/rtld.c:294: undefined reference to `_begin'

--- elf/Makefile.old	2009-11-28 02:02:16.000000000 +0600
+++ elf/Makefile	2009-11-28 02:02:27.000000000 +0600
@@ -300,7 +300,7 @@
 		  $(LDFLAGS-rtld) -Wl,-z,defs -Wl,--verbose 2>&1 |	\
 		  LC_ALL=C \
 		  sed -e '/^=========/,/^=========/!d;/^=========/d'	\
-		      -e 's/\. = 0 + SIZEOF_HEADERS;/& _begin = . - SIZEOF_HEADERS;/' \
+		      -e 's/\. = .* + SIZEOF_HEADERS;/& _begin = . - SIZEOF_HEADERS;/' \
 		  > $@.lds
 	$(LINK.o) -nostdlib -nostartfiles -shared -o $@			\
 		  $(LDFLAGS-rtld) -Wl,-z,defs $(z-now-$(bind-now))	\
